capstone/capapi/view_helpers.py

```diff
@@ -1,35 +1,30 @@
+import logging
```

Comment:
I have some bikeshedding comments below of the query filter stuff. Another general suggestion is to switch to DRF's built-in filtering classes instead of rolling our own for this. Perma has an example here:

https://github.com/harvard-lil/perma/blob/develop/perma_web/api/views.py#L28

We could add that as a bug for later if you don't want to mess with it now, though.